env/build.go

```diff
@@ -68,3 +82,30 @@ var POSIXBuildEnv = map[string][]string{
 		"PKG_CONFIG_PATH",
 	},
 }
+
+// removeFilter is generic function used to filter env vars out of the environment
+// if removeFilter(key) returns true, then the env var will be removed from the respective environment
+type removeFilter func(key string) bool
```

Comment:
Since this is only used by `isNotMember` now I wonder if we need it. Alternatively we could change the signature of `varsFromEnv` to accept this type.

Reply:
Removing for now.